src/api/app/controllers/webui/patchinfo_controller.rb

```diff
@@ -274,6 +223,67 @@ def new_tracker
     render json: { error: error, issues: issue_collection }
   end
 
+  private
+
+  def read_patchinfo
+    @binaries = []
```

Comment:
I believe this line and the following `each` loop could be done like this:
```ruby
@binaries = @file.map { |file| file.binary.text }
```

Reply:
I just saw in the previous PR that it's not possible as `@file` is a `ActiveXML::Node`.

Reply:
Sadly yes. Welcome to the ugly world of OBS:grimacing: